### PR TITLE
adding burger menu and navigation to about, help, update and contact …

### PR DESCRIPTION
**Pull request by kajaSlo**
…pages

### CODE REVIEW

**Comment by tciesek** [comment]
Please do not push your `.idea` folder to github. You can rebase your branch with latest change on master which have `.gitignore` that will take care of that.

**Comment by tciesek** [comment]
Why did you add gradle to `/app` folder? It should be only in the main folder.

File: app/src/main/java/com/grey/virtualfais/MainFragment.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        return view;
+    }
+
+    private void bind(View view) {
+        image = view.findViewById(R.id.image);
+        imageMask = view.findViewById(R.id.image_areas);
+    }
+
+    private void setupViews() {
+
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/MainFragment.java]: Why this method is empty? Maybe it should be removed

File: app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        bind(view);
+        setupViews();
+
+        return view;
+    }
+
+
+
+    private void bind(View view) {
+
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java]: Why this method is empty? Maybe it should be removed

File: app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    private void setupViews() {
+
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java]: Why this method is empty? Maybe it should be removed

File: app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_about, container, false);
+
+
+        bind(view);
+        setupViews();
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/about/AboutFragment.java]: These methods are empty so maybe these calls should be removed

File: app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        View view = inflater.inflate(R.layout.fragment_contact, container, false);
+
+        bind(view);
+        setupViews();
+
+        return view;
+    }
+
+    private void bind(View view) {
+
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java]: Why empty method?

File: app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    private void setupViews() {
+
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java]: Why empty method?

File: app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_contact, container, false);
+
+        bind(view);
+        setupViews();
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/contact/ContactFragment.java]: These two lines does nothing, I think....

File: app/src/main/java/com/grey/virtualfais/modules/help/HelpFragment.java [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_help, container, false);
+
+        bind(view);
+        setupViews();
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/help/HelpFragment.java]: The same as in the other files - calls to empty methods

File: app/src/main/java/com/grey/virtualfais/modules/update/UpdateFragment.java [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_update, container, false);
+
+        bind(view);
+        setupViews();
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/modules/update/UpdateFragment.java]: The same as in the other files - calls to empty methods

File: app/src/main/java/com/grey/virtualfais/MainActivity.java [hunk 10]
```diff
@@ -1,18 +1,250 @@
...
+
+        if (restored != null) {
+            currentFragment = restored;
+            runFragment();
+        }
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
```
Comment (magdalena-szkaradek) [inline app/src/main/java/com/grey/virtualfais/MainActivity.java]: This method return always false

**Comment by grey277** [comment]
I'm closing it, it is already merged.

**Pull request closed without merging**